portal-api/src/v1/users/routes.js

```diff
@@ -154,32 +155,67 @@ module.exports.findById = (req, res, next) => {
 };
 
 module.exports.updateById = (req, res, next) => {
-  findOne(req.params._id, (error, user) => {
-    if (error) {
-      next(error);
-    } else if (user) {
-      const errors = applyUpdateRules(user, req.body);
-      if (errors.length) {
-        res.status(400).json({
-          success: false,
-          errors: {
-            count: errors.length,
-            errorList: combineErrors(errors),
-          },
-        });
+  if (req.user.roles.includes(ADMIN)) {
+    findOne(req.params._id, (error, user) => {
+      if (error) {
+        next(error);
```

Comment:
Unless you are expecting to call something after `next(error)` from this controller, I would change this to `return next(error);` and then change the `else if` to a plain new `if` below